AutoInstaller.ps1

```diff
@@ -0,0 +1,45 @@
+#Requires -RunAsAdministrator
+
+# Check user has webhook URL ready
+$userPrompt = Read-Host -Prompt "Do you have your webhook URL ready? Y/N"
+
+# Prompt user to create webhook first if not ready
+If ($userprompt -ne 'Y') {
+    Write-Output "Please create a Discord webhook before continuing. `nFull instructions avalible at https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks"
+    exit
+}
+# Get latest release from GitHub
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+$latestRelease = Invoke-WebRequest -Uri https://github.com/tigattack/VeeamDiscordNotifications/releases/latest -Headers @{"Accept"="application/json"} -UseBasicParsing

```

Comment:
Can use `-ContentType` parameter instead of `-Headers` to set the content type of the request.
```suggestion
$latestRelease = Invoke-WebRequest -Uri https://github.com/tigattack/VeeamDiscordNotifications/releases/latest -ContentType 'application/json` -UseBasicParsing
```